settings.py

```diff
@@ -998,6 +998,8 @@
 DATADOG_API_KEY = None
 DATADOG_APP_KEY = None
 
+PROCESSOR_BACKEND = 'couch'  # either couch or postgres
```

Comment:
at first i thought we'd configure an entire instance based on couch or sql. in some ways this is nice because it gives us the ability to know things earlier. in other ways this massively sucks because we can't do this on a per user basis. in we want to go that route we should use thread locals which is what i'm leaning towards now